platform/android/export/export.cpp

```diff
@@ -2725,28 +2777,26 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 			unzCloseCurrentFile(pkg);
 
 			//write
-			if (!use_custom_build) {
```

Comment:
No need for this flag anymore since if `use_custom_build` is true, then `export_project` exits before it leaves the `if (use_custom_build)` branch.